client/my-sites/domains/domain-management/name-servers/index.jsx

```diff
@@ -299,7 +299,7 @@ class NameServers extends React.Component {
 				isPlaceholder={ this.isLoading() }
 				path={ domainManagementDns( this.props.selectedSite.slug, this.props.selectedDomainName ) }
 			>
-				{ this.props.translate( 'DNS Records' ) }
+				{ this.props.translate( 'DNS records' ) }
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **26** times:
`translate( 'New DNS records saved.' )` *ES Score: 9*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/more-button-sentence-casing)